frontend/src/components/worksheets/items/ItemWrapper.js

```diff
@@ -59,11 +59,15 @@ class ItemWrapper extends React.Component {
             isWorkSheetItem = false;
         }
 
+        const {isDummyItem} = item;
```

Comment:
these two variables don't seem like they type check

Reply:
What do you mean? This is the same as saying

`const isDummyItem = item.isDummyItem;`

Reply:
Oh, misread, never mind.